WalletWasabi/Crypto/ZeroKnowledge/LinearRelation/Equation.cs

```diff
@@ -0,0 +1,61 @@
+using NBitcoin.Secp256k1;
+using System.Linq;
+using WalletWasabi.Crypto;
+using WalletWasabi.Crypto.Groups;
+using WalletWasabi.Helpers;
+
+namespace WalletWasabi.Crypto.ZeroKnowledge.LinearRelation
```

Comment:
this is in its own namespace under `ZeroKnowledge` because the corresponding `Statement` and `Knowledge` classes will be added in the next pull request